jsons/Buildings.json

```diff
@@ -204,7 +204,8 @@
         "happiness": 2,
         "uniques": [
             "Unsellable", "Unbuildable",
-            "[+2] population [in this city]"
+            "[+2] population [in this city]",
+            "[-25]% Culture cost of natural border growth [in this city]"
```

Comment:
Is this to make up for the missing extra tiles it starts with? or is this an actual extra of the building?

Reply:
Yea, tried to replace 6 extra tiles at the foundation city...

Reply:
This is a problem elsewhere as well. This should be done as a request for a new Unciv unique

Reply:
True, but this mock is quite harmless, I can note in the comment that it needs to be reworked when the functionality appears

Reply:
Ehhhh... I'm not super convinced it's so harmless, especially given how I've seen it impact Shoshone. Add in the intended effect (boost a later city to a level closer to where it would be later in a game), and I'd rather get a proper unique for this

Reply:
Okay, you've convinced me. I'd like to see this mechanic in the future. Maybe it would be possible to give a discount on the purchase of cells?
I've cleaned it up so far.

Reply:
I found something. 
"Gain control over [Water?] tiles in a [2]-tile radius"

Reply:
Still doesn't do exactly what is wanted here

Reply:
cancelled. 0d037466c235f34dfda3b3b5be8555bd84e0d9bf